cli/wallet/wallet.go

```diff
@@ -206,11 +206,15 @@ func NewCommands() []cli.Command {
 			{
 				Name:      "remove",
 				Usage:     "remove an account from the wallet",
-				UsageText: "remove --wallet <path> [--force] <addr>",
+				UsageText: "remove --wallet <path> [--force] --account <addr>",
```

Comment:
Breaking change, although it's not a frequently used command, so probably can afford it.

Reply:
It's just for unification. Do we have to be compatible with 2x cli?

Reply:
Also, we use `--address` for such options.

Reply:
2.x is irrelevant, it's just that we already have some scripts written for current syntax, so we can't arbitrarily change CLI. At the same time, this command shouldn't be used by that scripts, so it's safe to change.

Reply:
Fixed.